.circleci/config.yml

```diff
@@ -6,6 +6,11 @@ anchors:
     docker:
       - image: circleci/node:8
 
+  - &default_mac
+    working_directory: /Users/distiller/project
+    macos: # https://circleci.com/docs/2.0/testing-ios/#supported-xcode-versions
+      xcode: "9.4.1"
```

Comment:
We need 10.0.0 now for iOS 12 (specifically if you're developing on a phone with iOS 12), can you see if CI passes with the latest Xcode?

Reply:
I can check that, but we've had problems in the past when we compile the shell apps with a later version of xcode than turtle has.  I didn't realize we needed 10.0 soon.  If so, then I'll need to make a new turtle vm image.

Reply:
It's not urgent but in the past we've seen issues when people install simulator IPAs on mismatched versions of the iOS simulator, so as more people upgrade to Xcode 10 we'll want to match them.